src/main/java/com/yugabyte/sample/apps/SqlInserts.java

```diff
@@ -77,15 +77,24 @@ public void createTablesIfNeeded() throws Exception {
     Connection connection = getPostgresConnection();
 
     // Check if database already exists.
-    connection.createStatement().execute(
-      String.format("CREATE DATABASE %s", postgres_ybdemo_database));
+    ResultSet rs = connection.createStatement().executeQuery(
+        String.format("SELECT datname FROM pg_database WHERE datname='%s'",
```

Comment:
Instead of trying this select without a create, let's change the original statement to:
```CREATE DATABASE %s IF NOT EXISTS```
I think the YCQL side of the code is laid out that way.